packages/manager/modules/billing/src/autoRenew/actions/terminate-all-dom/routing.js

```diff
@@ -0,0 +1,34 @@
+import map from 'lodash/map';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.account.billing.autorenew.terminate-all-dom', {
+    url: '/delete-all-dom?serviceId&serviceType',
+    views: {
+      modal: {
+        component: 'billingAutorenewTerminateAllDom',
+      },
+    },
+    layout: 'modal',
+    translations: { value: ['.'], format: 'json' },
```

Comment:
Is this required ? can be replaced with `.run(/* @ngTranslationsInject:json ./translations */);` in module.js